webview-ui/src/components/ResolutionsPage/ResolutionsPage.tsx

```diff
@@ -104,73 +93,52 @@ const ResolutionPage: React.FC = () => {
                   />
                 </ChatCard>
               </FlexItem>
-              <FlexItem>
-                <YellowLabel>Please provide resolution for this issue.</YellowLabel>
-              </FlexItem>
+              <SentResponse>Please provide resolution for this issue.</SentResponse>
             </Flex>
           )}
+
           <Flex
-            direction={{
-              default: "column",
-            }}
+            direction={{ default: "column" }}
             grow={{ default: "grow" }}
             alignItems={{ default: "alignItemsFlexStart" }}
           >
-            {hasNothingToView && (
-              <FlexItem>
-                <Label color="blue">No resolutions available.</Label>
-              </FlexItem>
-            )}
+            {hasNothingToView && <ReceivedResponse>No resolutions available.</ReceivedResponse>}
             {isHistorySolution && (
-              <FlexItem>
-                <Label color="blue">Loaded last known resolution.</Label>
-              </FlexItem>
+              <ReceivedResponse>Loaded last known resolution.</ReceivedResponse>
             )}
             {solutionMessages.map((msg) => (
-              <FlexItem key={msg}>
-                <Label color="blue">{msg}</Label>
-              </FlexItem>
+              <ReceivedResponse key={msg}>{msg}</ReceivedResponse>
             ))}
             {isFetchingSolution && <Spinner />}
 
             {hasResponse && (
-              <FlexItem>
-                <ChatCard color="blue">
-                  <FileChanges
-                    changes={getRemainingFiles()}
-                    onFileClick={handleFileClick}
-                    onApplyFix={handleAcceptClick}
-                    onRejectChanges={handleRejectClick}
-                  />
-                </ChatCard>
-              </FlexItem>
+              <ReceivedResponse>
```

Comment:
Bigger objects look really bad inside Label. Actually I consider a Label a temporary approach until we figure out a better way to display the messages.